e2e/e2e.go

```diff
@@ -148,15 +148,19 @@ func grabDependencies(ctx context.Context, deps map[string]bool, name string, au
 }
 
 func parseServiceYaml() (*Service, error) {
```

Comment:
Nit: Should the `Service` type be renamed?

Reply:
i left it alone because i figured this should honestly come from a package in `devenv` whenever you finish that ticket you have opened

Reply:
the one for doing this in the devenv natively

Reply:
i would've changed it too if you wanted me to, i just had auto-merge enabled :/

Reply:
meh, you're right this will go away when devenv get the implementation